drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -492,6 +511,7 @@ function DragAndDropEditBlock(runtime, element, params) {
                         var data = {
                             'display_name': $element.find('#display-name').val(),
                             'mode': $element.find("#problem-mode").val(),
+                            'max_attempts': $element.find("#max-attempts").val(),
```

Comment:
I recommend that you use classes instead of ids-- it is a general best practice to avoid ids.